### DISCUSSION TITLE
特定のキーが入力できない@keyball61

### DISCUSSION THREAD

**Discussion by hina1322** [Yowkees/keyball#96] [Q&A] [answered]
右トラックボールのdefaultプロファイルを使用しています．
反応しないキーは以下です．(PCBシルク)
```
左手側
7，11，13，16

右手側
15
```
上記スイッチにおいて，スイッチ押下時及びダイオードの導通確認はしました．
上記以外のスイッチは，oledモジュールへの表示レベルで機能していることを確認済みです．

どのような観点で確認していけばよろしいでしょうか?
↑確認済み項目についても，もう一度確認すべきことがありましたらご教示ください

**Comment by Yowkees** [comment, marked as answer]
結構反応しないキーが多いですね💦
原因としてよく聞くのがキースイッチの足折れです。一度該当のキースイッチを外してご確認ください。
問題なければ半田付け不良が考えられますので、半田を追加で盛ってみてご確認お願いします。

**Reply by hina1322** [reply]
ソケットのはんだ付け不良でした．．．
無事すべてのスイッチで入力確認できました!
お手数おかけしました
